app/public/components/rct/rctData/pagesCellsSpecials.js

```diff
@@ -94,10 +94,10 @@ pagesCellsSpecials[pagesNames.dataPasses] = {
 };
 pagesCellsSpecials[pagesNames.mc] = {};
 
-const dateForametter = (sec) => {
-    const d = new Date(Number(sec));
-    const expected = -120;
-    d.setMinutes(d.getMinutes() - (expected - d.getTimezoneOffset()));
+const dateFormatter = (sec) => {
+    const cestOffset = 2 * 60 * 60 * 1000;
+    const localOffset = new Date().getTimezoneOffset() * 60 * 1000;
+    const d = new Date(Number(sec) + cestOffset + localOffset);
```

Comment:
Creating a new date from seconds seems more bug proof :bug:
External libraries basically do the exact same thing, so I do not see the point of depending on them instead of writing these three lines on our own :shrug: